etc/pip_install/toree/toreeapp.py

```diff
@@ -57,6 +59,12 @@ class ToreeInstall(InstallKernelSpec):
     spark_home = Unicode(os.getenv(SPARK_HOME, '/usr/local/spark'), config=True,
```

Comment:
In general, do we actually want to use default values here? I am assuming we don't really have a standard default place to deploy Spark/Hadoop and maybe it would be better to use the env variables if they are available, otherwise, ignore or in case of required ones throw an error?

Reply:
If they're set to be empty, it's just like they're not set -- so it's true that probably they're better empty.